### DISCUSSION TITLE
Suddenly Git plugin gives an error "exceeded its quota limitations" only in Firefox

### DISCUSSION THREAD

**Discussion by bigdiff** [floccusaddon/floccus#1679] [Q&A] [answered]
Approximately four days ago, I noticed that my bookmarks were not syncing properly. Upon investigating the logs, I encountered the following error message (attached below).

Interestingly, this limitation seems to be specific to Firefox, as other browsers do not exhibit the same behavior. My hypothesis is that this issue arose after updating to Firefox version 128.0, as the problem coincided with the update. Despite attempting to re-sync, the bookmarks remain unsynchronized.

<details>
<summary>Logs</summary>

```
Starting sync process for account bookmarks.xbel
onSyncStart: begin
(git) init
onSyncFail
Syncing failed with The current transaction exceeded its quota limitations.
onSyncFail
```

</details>

**Comment by bigdiff** [comment]
It could be due to some MV3 rollout they're currently working on.

**Comment by marcelklehr** [comment, marked as answer]
Mmmh, does it help to reinstall floccus?

**Reply by bigdiff** [reply]
yeah that seems to have fixed the issue. thank you :pray:

**Comment by marcelklehr** [comment]
It seems the git backend leaks indexedDB databases. This will be fixed in the next release.
